include/Dialect/LWE/IR/LWEAttributes.td

```diff
@@ -267,6 +267,8 @@ def LWE_LWEParams : AttrDef<LWE_Dialect, "LWEParams"> {
   let parameters = (ins "IntegerAttr": $cmod, "unsigned":$dimension);
 
   let assemblyFormat = "`<` struct(params) `>`";
+
+  let isOptional = 1;
```

Comment:
The docs on this say

```
  // Whether the attribute is optional. Typically requires a custom
  // convertFromStorage method to handle the case where the attribute is
  // not present.
```

Do we need that?

Reply:
Oops, nope - I was playing around with when exploring the codegen for optional attrs, but nope - the op that uses that (trivial encrypt) where it may be optional wraps it in `OptionalAttr`.